.drone.yml

```diff
@@ -85,7 +85,7 @@ pipeline:
     group: docker
     repo: plugins/codecov
     auto_tag: true
-    auto_tag_suffix: amd64
+    auto_tag_suffix: linux-amd64
```

Comment:
Can we assume that without an `-os-` suffix, it's for linux?
This would keep in line with the previous tags and other plugins, and reduce verbosity.

```
# linux
plugins/codecov:1.1.1-amd64
plugins/codecov:1.1.1-arm
plugins/codecov:1.1.1-arm64
plugins/codecov:1.1.1-i368

# windows
plugins/codecov:1.1.1-windows-amd64
plugins/codecov:1.1.1-windows-arm
plugins/codecov:1.1.1-windows-arm64
plugins/codecov:1.1.1-windows-i368

Reply:
Brad asked for it at https://github.com/drone-plugins/drone-codecov/issues/7#issuecomment-359917440, that's why I added it, in a future plugins/manifest version I will also ad auto_tag to the "from-args" command, than we can even drop the manifest.tmpl.

Reply:
Gotcha, thanks. Won't argue there :)

Reply:
So will there still be a `latest` tag, or is that absorbed into `plugins/codecov:linux-amd64` and `plugins/codecov:windows-amd64` (the semver-less tags)?

Reply:
The latest tag will be automatically set by the manifest tool, this unified the image name also for multi arch setups

Reply:
But yes, this comes out from the docker plugin auto tagging